ui.apps/src/main/content/jcr_root/apps/etoolbox-rollout-manager/clientlibs/rollout-manager-ui/js/console-ui.actions.js

```diff
@@ -25,16 +25,13 @@
      * Retrieves data related to eligible for synchronization live copies as a json array. The data is
      * used for building 'Targets' tree in the UI dialog
      * @param path - path of the page selected in Sites
-     * @returns {*}
+     * @returns {Promise}
```

Comment:
Cosmetic, currently it is actually
```suggestion
     * @returns {JQuery.Promise}
```